plugins/inputs/elasticsearch/elasticsearch.go

```diff
@@ -259,13 +290,17 @@ func (e *Elasticsearch) gatherNodeStats(url string, acc telegraf.Accumulator) er
 
 		now := time.Now()
 		for p, s := range stats {
+			if !e.individualStatIncluded(p) {
```

Comment:
I still don't understand why this is done.  It seems to me that the stats not included won't be in the slice we are ranging over and so this will never be hit.

Reply:
We are ranging over the property-to-string-mapping `stats`. I cannot think of a better way of dynamically mapping the names. We go from snake case to pascal case back to snake case. Is there a way to re-use the original JSON mappings (JSON to struct) to get the snake case names back. We are defining these mappings twice right now...

I added a simple `nil` check for now.